mantidproject/mantid
# Enter exact positions in MultiSlice view

Description of work.

This provides a context menu for manually editing the position of slices. The `shift`, `control` and `none` modifiers are already in use, so I selected `alt`.  

**To test:**



1. Open a MDWorkspace in Mantid
2. Right click and open in the VSI.
3. (If necessary) switch to the MultiSlice view
4. Press alt while clicking on one of the axis arrows. Verify that a text box appears.
5. Try entering an invalid input. Verify that most letters or out-of-range values are not allowed and turn the window yellow.
6. Try entering a valid input. Verify that the slice position changes to the expected position.

There is no GitHub issue associated with this pull request

**Release Notes** 

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
